Library/Homebrew/cmd/update.sh

```diff
@@ -9,22 +9,42 @@ git() {
 git_init_if_necessary() {
   if [[ -n "$HOMEBREW_OSX" ]]
   then
-    OFFICIAL_REMOTE="https://github.com/Homebrew/brew.git"
+    BREW_OFFICIAL_REMOTE="https://github.com/Homebrew/brew.git"
+    CORE_OFFICIAL_REMOTE="https://github.com/Homebrew/homebrew-core"
```

Comment:
We're just being inconsistent I guess. :wink: `brew tap` always omits the `.git`, the migration from `legacy-homebrew` to `brew` also uses `https://github.com/Homebrew/brew.git` while our install script uses `https://github.com/Homebrew/brew` (so that's one way to differentiate new users from those that have been migrated).